frontend/src/components/AddNewTripComponents/AddNewTrip.tsx

```diff
@@ -0,0 +1,216 @@
+            case 4: {
+                return (
+                    <form onSubmit={onClickNext}>
+                        <AddShoppingInfo setAmountOfClothingItems={setAmountOfClothingItems}
+                                         amountOfClothingItems={amountOfClothingItems}
+                                         amountOfElectronicItems={amountOfElectronicItems}
+                                         setAmountOfElectronicItems={setAmountOfElectronicItems}
+                                         amountOfCustomShoppingItem={amountOfCustomShoppingItem}
+                                         setAmountOfCustomShoppingItem={setAmountOfCustomShoppingItem}
+                                         customShoppingItem={customShoppingItem}
+                                         setCustomShoppingItem={setCustomShoppingItem}
+                                         customShoppingItemEmission={customShoppingItemEmission}
+                                         setCustomShoppingItemEmission={setCustomShoppingItemEmission}
+                                         amountOfSouvenirItems={amountOfSouvenirItems}
+                                         setAmountOfSouvenirItems={setAmountOfSouvenirItems}/>
+                        <button type={"button"} className={"next"} onClick={onClickNext}>Next</button>
+                        <button type={"button"} className={"return"} onClick={onClickReturn}>Return</button>
+                    </form>
+                )
+            }
+            case 5:
+                return (
+                    <form onSubmit={onAdd}>
+                        <AddActivityInfo amountOfGolfRounds={amountOfGolfRounds}
+                                         setAmountOfGolfRounds={setAmountOfGolfRounds}
+                                         amountOfSkiingDays={amountOfSkiingDays}
+                                         setAmountOfSkiingDays={setAmountOfSkiingDays}
+                                         amountOfBeautyDays={amountOfBeautyDays}
+                                         setAmountOfBeautyDays={setAmountOfBeautyDays}
+                                         customActivityItem={customActivityItem}
+                                         setAmountOfCustomActivityItem={setAmountOfCustomActivityItem}
+                                         customActivityItemEmission={customActivityItemEmission}
+                                         setCustomActivityItem={setCustomActivityItem}
+                                         amountOfCustomActivityItem={amountOfCustomActivityItem}
+                                         setCustomActivityItemEmission={setCustomActivityItemEmission}/>
+                        <button type={"button"} className={"return"} onClick={onClickReturn}>Return</button>
+                        <button className={"add-trip"}>Add Trip</button>
+                    </form>
+                )
+
+            default: {
+                return (<div>"Fehler"</div>)
```

Comment:
Hier ist noch ein deutscher Ausdruck. Sonst ist alles Englisch.
```suggestion
                return (<div>"Error"</div>)
```